### PR TITLE
S15 Suggested products

### PR DESCRIPTION
**Pull request by Zeroya**

### CODE REVIEW

File: src/components/ProductSuggestList/ProductSuggestList.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                    <button className={s.card__button}>Buy now</button>
+                  </Link>
+                </div>
+              </div>
+            ))}
+      </div>
+    </div>
+  );
+};
+
```
Comment (Kost927) [inline src/components/ProductSuggestList/ProductSuggestList.tsx]: again huge JSX

File: src/components/ProductSuggestList/ProductSuggestList.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          <GreenMark />
+        </button>
+      </div>
+      <div className={s.suggestList__cards}>
+        {!suggestList?.length && (
+          <Alert message="No similar product found, sorry, keep searching other products" type="info" showIcon />
+        )}
+        {suggestList &&
+          JSON.parse(JSON.stringify(suggestList))
+            ?.splice(0, counter)
```
Comment (Kost927) [inline src/components/ProductSuggestList/ProductSuggestList.tsx]: leave only map. Logic in JSX
const XXX = suggestList &&
          JSON.parse(JSON.stringify(suggestList))
            ?.splice(0, counter);
XXX.map()

File: src/components/ProductSuggestList/ProductSuggestList.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  }, [params.productId]);
+
+  return (
+    <div className={s.suggestList}>
+      <div className={s.suggestList__intro}>
+        <h1 className={s.suggestList__title}>You will maybe love</h1>
+        <button
+          onClick={addSuggestProduct}
+          className={`${s.suggestList__button} ${
+            (suggestList?.length < 4 || suggestList?.length - 1 < counter) && s.suggestList__button_disable
```
Comment (Kost927) [inline src/components/ProductSuggestList/ProductSuggestList.tsx]: logic in JSX

File: src/components/ProductSuggestList/ProductSuggestList.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                    alt={`${card.name} photo`}
+                  />
+                </div>
+                <div className={s.card__info}>
+                  <div className={s.card__title}>{firstLetterStrUpperCase(card.name)}</div>
+                  <div className={s.card__description}>{card.description.slice(0, 40).trim().concat("...")}</div>
+                </div>
+                <div className={s.card__priceBlock}>
+                  <div className={s.card__price}>
+                    <h4>{calcDiscountPrice(card.price, card.discount)} USD</h4>
```
Comment (Yuliia143) [inline src/components/ProductSuggestList/ProductSuggestList.tsx]: Should be bold, and seems it shouldn't be h4

**Review by Yuliia143 — APPROVED** [review]

**Pull request merged**